### DISCUSSION TITLE
是否可以实现发起群聊功能？

### DISCUSSION THREAD

**Discussion by li1553770945** [howmanybots/onebot#234] [想法 / 建议]
想要做一个类似于Oncall的机器人，主要功能为：用户添加机器人为好友，向机器人发送特定问题，机器人拉用户和特定的几位工作人员创建一个群聊，在这个群里解决问题。问题解决后，由bot解散群聊。
目前其他功能都支持了，最大的问题是不支持主动发起群聊功能，请问是否可以支持呢？

**Comment by ExerciseBook** [comment]
我感觉 onebot 里可以加上这个 action，不知道其他大活儿是什么想法？

**Reply by stdrc** [reply]
我觉得得先有实现支持，再在标准里加才有意义（
